code/game/objects/items.dm

```diff
@@ -244,6 +244,9 @@
 	if(flags_item & DELONDROP)
 		qdel(src)
 
+///Called whenever an item is unequipped to a new loc (IE, not when the item ends up in the hands)
+/obj/item/proc/removed_from_inventory(mob/user)
```

Comment:
that happens when dropped ? and it sends a signal ?

Reply:
![Capture](https://user-images.githubusercontent.com/25491240/104316172-5fb27000-54dc-11eb-9ede-7e62cddb09d6.PNG)
Happens here